### PR TITLE
feat(i18n): pluginpage component

### PR DESCRIPTION
**Pull request by TikWind**
i18n pluginpage component

### CODE REVIEW

File: src/components/PluginPage/locales/en-US.ts [hunk 1]
```diff
@@ -1,163 +1,172 @@
 export default {
-  'PluginForm.plugin.limit-conn.desc': 'Limit the number of concurrent connections',
-  'PluginForm.plugin.limit-conn.property.conn': 'conn',
-  'PluginForm.plugin.limit-conn.property.conn.extra': 'Maximum number of concurrent connections',
-  'PluginForm.plugin.limit-conn.property.burst': 'burst',
-  'PluginForm.plugin.limit-conn.property.burst.extra':
+  'pluginpage.plugin.limit-conn.desc': 'Limit the number of concurrent connections',
+  'pluginpage.plugin.limit-conn.property.conn': 'conn',
+  'pluginpage.plugin.limit-conn.property.conn.extra': 'Maximum number of concurrent connections',
```
Comment (juzhiyuan) [inline src/components/PluginPage/locales/en-US.ts:4]: Please use PluginPage instead.

Comment (TikWind) [reply src/components/PluginPage/locales/en-US.ts:4]: Can I use lower case ?

Comment (juzhiyuan) [reply src/components/PluginPage/locales/en-US.ts:4]: Better not, we may obey the `ModuleName.key1.key2` style.

Comment (TikWind) [reply src/components/PluginPage/locales/en-US.ts:4]: The demo in umijs maybe all lower case.

Comment (juzhiyuan) [reply src/components/PluginPage/locales/en-US.ts:4]: pluginpage is grouped by plugin with page, we need some ways to let developers know clearly what it is.

Comment (TikWind) [reply src/components/PluginPage/locales/en-US.ts:4]: OK

Comment (TikWind) [reply src/components/PluginPage/locales/en-US.ts]: I have changed "pluginpage" to "PluginPage".

**Pull request merged**